controller/lib/core/sensors.h

```diff
@@ -48,6 +50,22 @@ class Sensors {
   // are called.
   void Calibrate();
 
+  // Called once for every breath cycle.  Sensors assume that tidal volume
```

Comment:
Shouldn't these changes be in TVIntegrator? I initially added it cause I was assuming that drift correction would live there. If it won't, then maybe just inline that class.

Reply:
Hm, this would be really nice, if only because TVIntegrator is easier to unit test than the full Sensors module.

For the purpose of drawing a flow graph that isn't terrible, we need to adjust flow too.  Basically I'm using TV integration to adjust the flow.  I guess I could have TVIntegrator have a flow_offset variable -- does that seem weird to you?  Leaving open for your thoughts.

Reply:
I think that means that TVIntegrator should be renamed to FlowIntegrator and it should compute both the correct flow and the correct integral thereof. Up to you whether you want to do that in this PR or followup or not at all.

Reply:
Will make a note to do this in a followup.